README.md

```diff
@@ -1,3 +1,32 @@
+
 # pardus-image-writer
 
-a Python GTK App to burn ISO files on USB Flash sticks.
\ No newline at end of file
+A tool for writing image files to removable storage devices such as USB drives and SD cards.
+
+# Introduction
+
+pardus-image-writer is a simple and user-friendly tool for writing image files to removable storage devices. It's designed to be fast, reliable, and easy to use, making it the perfect solution for anyone who needs to write images to removable storage quickly and efficiently.
+
+# Features
+- Cross-platform compatibility: pardus-image-writer is written in Python, works on many linux distributions.
+- User-friendly interface: pardus-image-writer features a graphical user interface that's easy to use and understand, even for those who are new to image writing tools.
+- Works with both Windows ISOs and Linux ISOs: pardus-image-writer is capable of writing both Windows and Linux image files to removable storage devices, making it a versatile solution for a wide range of use cases.
+
+# Dependencies
+
+To use pardus-image-writer, you can use the following command to install necessary dependencies:
+
+```bash
+pip install setuptools pyudev
+```
+
+# Installation
+
+To install pardus-image-writer, follow these steps:
```

Comment:
Installing from .deb files is easy to handle for end users.

These steps can be written as a second alternative.

Please add these steps above:
```suggestion
1. Download the latest version of .deb file from [Releases](https://github.com/pardus/pardus-image-writer/releases) page.
2. Install the .deb file with Pardus Package Installer by double clicking on it. (or use `sudo apt install pardus-image-writer_0.4.0.Beta1_all.deb`)
```